src/facade/dragonfly_connection.cc

```diff
@@ -37,6 +37,9 @@ ABSL_FLAG(std::string, admin_bind, "",
           "This supports both HTTP and RESP "
           "protocols");
 
+ABSL_FLAG(std::uint64_t, request_cache_limit, 1ULL << 26,  // 64MB
+          "Amount of memory to use for request cache in bytes - per IO thread.");
+
```

Comment:
I thought we had those postifxes for memory sizes 😄

Reply:
what do you mean @dranikpg ?